frontend/src/components/vestigingscijfers/schooladviezen/VestigingscijferSchoolAdvies.vue

```diff
@@ -36,11 +39,13 @@ export default {
     async getData () {
       if (this.id) {
         let url = API_HOST + `/onderwijs/api/aggregated-advies/?vestiging=${this.id}`
-
-        let data = await readData(url)
-        order(data, 'advies', ['pro', 'vmbo b / k', 'vmbo g / t', 'havo / vwo'])
+        let data = order(await readData(url), 'advies', ['pro', 'vmbo b / k', 'vmbo g / t', 'havo / vwo'])
```

Comment:
SCHOOL_TYPES in config somewhere?

Reply:
will look into it, not relevant to OP-199